src/main/java/com/faforever/client/game/GamesTilesContainerController.java

```diff
@@ -65,7 +62,7 @@ public GamesTilesContainerController(UiService uiService, PreferencesService pre
     };
 
     gameListChangeListener = change -> {
-      Platform.runLater(() -> {
+      JavaFxUtil.assertApplicationThread();
```

Comment:
run Later made certain actions to be executed in the wrong order and it is on the Application thread anyways because game updates are only done on the application thread

Reply:
Race conditions caused a malfunction in checkboxes or them having no effect